src/consts/allowed-attributes.const.ts

```diff
@@ -15,7 +15,8 @@ export const ALLOWED_ATTRIBUTES: XSSWhiteList = {
     'data-video-href',
     'data-proposal',
     'class',
-    'title'
+    'title',
+    'id'
```

Comment:
Enabling `id` opens up some xss attack vectors, we shouldn't enable this attribute.
Instead, we can preprocess `id` attribute and create new attribute like `data-id` just like other attributes (e.g. `data-video-href`) we created. And then inside vision or mobile codebase, we can handle those `data-id` accordingly.